homeassistant/components/sia/const.py

```diff
@@ -0,0 +1,29 @@
+"""Constants for the sia integration."""
+
+from datetime import timedelta
+
+from homeassistant.components.alarm_control_panel import (
+    DOMAIN as ALARM_CONTROL_PANEL_DOMAIN,
+)
+from homeassistant.components.binary_sensor import DOMAIN as BINARY_SENSOR_DOMAIN
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
+
+CONF_ACCOUNT = "account"
+CONF_ACCOUNTS = "accounts"
+CONF_ADDITIONAL_ACCOUNTS = "additional_account"
+CONF_PING_INTERVAL = "ping_interval"
+CONF_ENCRYPTION_KEY = "encryption_key"
+CONF_ZONES = "zones"
+ABORT_ALREADY_CONFIGURED = "already_configured"
+DOMAIN = "sia"
+DATA_UPDATED = f"{DOMAIN}_data_updated"
+DEFAULT_NAME = "SIA Alarm"
+DEVICE_CLASS_ALARM = "alarm"
+HUB_SENSOR_NAME = "last_heartbeat"
+HUB_ZONE = 0
+PING_INTERVAL_MARGIN = timedelta(seconds=30)
+PREVIOUS_STATE = "previous_state"
+UTCNOW = "utcnow"
+LAST_MESSAGE = "last_message"
+
+PLATFORMS = [SENSOR_DOMAIN, BINARY_SENSOR_DOMAIN, ALARM_CONTROL_PANEL_DOMAIN]
```

Comment:
Please only add a single platform when adding a new integration.

Reply:
I had a good look at the logic here, and binary sensor can be deleted for now, so I did, but the way this protocol works is that you open a port, that port listens for messages from one or more alarm systems (also known as accounts, which comes back in another comment) and each alarm system can have multiple zones. But there is a zone=0 that sends a heartbeat for that system, which is represented by the Sensor platform, while the alarms are represented by the Alarm Control panel. since it is a passive protocol, so being able to see when the last heartbeat arrived gives you confidence that all is well.

Reply:
Understood, but the guideline is to only have one platform in the initial PR for an integration. You can always add the additional platforms in a future PR.

Reply:
Got rid of another platform for this initial PR, so inline with guidance!